modular_np_lethal/implants/code/razorwire.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/melee/razorwire
```

Comment:
This entire file we don't have anymore because I brought it upstream, your local might be out of date?

Reply:
Having trouble updating things. It doesn't seem to get deleted when I update the branch, I could just remove the entire file but I assume that uh. Won't work.